sdk/Lusid.Sdk.Tests/tutorials/Instruments/FutureExamples.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class FutureExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
+        {
+        }
+
+        internal override LusidInstrument CreateExampleInstrument() => InstrumentExamples.CreateExampleFuture();
+
+        internal override void GetAndValidatePortfolioCashFlows(LusidInstrument instrument, string scope, string portfolioCode, string recipeCode, string instrumentID)
```

Comment:
Is this ever called? Or do you need to add an example to this class that actually makes use of it?

Reply:
Think next iteration also (I'm not too familiar with it to get it working). This method would get used eventually :)